utilities/pipelines/staticValidation/module.tests.ps1

```diff
@@ -1643,12 +1643,12 @@ Describe 'API version tests' -Tag 'ApiCheck' {
 
         $approvedApiVersions = @()
         if ($AllowPreviewVersionsInAPITests) {
-            # We allow the latest 5 including previews (in case somebody wants to use preview), or the latest 3 non-preview
+            # We allow the latest 5 including previews (in case somebody wants to use preview), or the latest 5 non-preview
```

Comment:
Anything to be added to the wiki? ✍️

Reply:
As far as I can see, at the only place where the number of API versions is referenced outside of the test file, it already says `5`. So no ;)